### ISSUE TITLE
Dont set ROCR_VISIBLE_DEVICES and HIP_VISIBLE_DEVICES

### ISSUE THREAD

**Issue by IMbackK** [mcmonkeyprojects/SwarmUI#566]
### Expected Behavior

ROCR_VISIBLE_DEVICES ids and HIP device are redundant and dont refer to the same ids, ROCR uses the order supplied by the kernel drm interface while HIP uses the init order from rocr, which need not be the same as rocr will initialize gpus of the same architecture together

thus 
```
export ROCR_VISIBLE_DEVICES=1
export HIP_VISIBLE_DEVICES=1
```

can end up with _no_ device available since you have masked away all gpus

i would recommend using only  ROCR_VISIBLE_DEVICES as this works in all cases. EDIT: besides windows maybe? not sure.

### Steps to Reproduce

Have a machine with n gfx908 gpus and z gfx1030 gpus, be unlucky with the initialization order and try 

```
$ export ROCR_VISIBLE_DEVICES=1
$ export HIP_VISIBLE_DEVICES=1
$ python
>>> import torch
>>> torch.cuda.is_available()
False
```

### Debug Logs

N/A

**Comment by IMbackK** [comment]
Alternatively you could set the second id to the gpu uid of the first. like export ROCR_VISIBLE_DEVICES=GPU-f980f88ef31631a2

**Comment by mcmonkey4eva** [comment]
Full disclosure I have no idea how rocm/hip work and don't have any environment to test in, so I'm just going to trust you on that. I've edited it to exclude hip on Linux (but leave it on windows for now at least)
